mantle/cmd/kola/options.go

```diff
@@ -164,6 +164,11 @@ func syncOptionsImpl(useCosa bool) error {
 		kolaPlatform = "qemu-unpriv"
 	}
 
+	// native 4k requires a UEFI bootloader
+	if kola.QEMUOptions.Native4k && kola.QEMUOptions.Firmware == "bios" {
```

Comment:
Hmm, although we only want to do this automatic conversion if users didn't explicitly set `--qemu-firmware=bios`. If that's the case, we should error out. (We didn't care about this in the `cmd-run` code because there was no explicit `--bios` switch).

Reply:
Is it really worth trying to error out for that?

Reply:
I think it's worth covering, but it's clearly not a blocker.